gnome-shell/src/gnome-shell-sass/_colors.scss

```diff
@@ -34,7 +34,7 @@ $tooltip_borders_color: $osd_outer_borders_color;
 $shadow_color: transparentize(black, 0.9);
 
 //insensitive state derived colors
-$insensitive_fg_color: mix($fg_color, $bg_color, 50%);
+$insensitive_fg_color: mix($fg_color, $bg_color, 70%);
```

Comment:
Oh man, this happens if you do too much in one time. Did not want to commit this @clobrano , sorry. 
WIll make a PR for this specific line later